src/views/BottomTabBar.tsx

```diff
@@ -457,12 +457,14 @@ class TabBarBottom extends React.Component<BottomTabBarProps, State> {
                     }),
                   },
                 ],
-                // Absolutely position the tab bar so that the content is below it
-                // This is needed to avoid gap at bottom when the tab bar is hidden
-                position: this.state.keyboard ? 'absolute' : null,
               }
             : null,
           containerStyle,
+          {
+            // Absolutely position the tab bar so that the content is below it
+            // This is needed to avoid gap at bottom when the tab bar is hidden
+            position: this.state.keyboard ? 'absolute' : undefined,
```

Comment:
Thanks for the PR.

This isn't the correct place to put it. Coz we are updating `this.state.keyboard` regardless `keyboardHidesTabBar` enabled or not and we won't want to set position to `absolute` when this setting is not enabled.

I think the bug is in the ordering of styles. Moving `containerStyle` before the style related to `keyboardHidesTabBar` should fix this. Can you update the PR with the change?

Reply:
@satya164 that's right, thanks. I reordered those styles and tested with `keyboardHidesTabBar` true / false; it looks good.